Rules/CommonScripts/KillMessages.as

```diff
@@ -310,6 +333,47 @@ void onTick(CRules@ this)
 	{
 		feed.Update();
 	}
+
+	// display message in chat whenever a killstreak happens
+	for(int a = 0; a < getPlayerCount(); a++) 
+	{ 
+		CPlayer@ player = getPlayer(a);
+		CBlob@ blob = player.getBlob();
+
+		if (blob !is null && player !is null && blob.exists("killstreak"))
+		{
+			if(getGameTime() - blob.get_u32("kill time") > 180 && blob.get_u8("killstreak") > 4)
+			{
+				string multiKill;
+
+				switch (blob.get_u8("killstreak"))
+				{
+					case 5: multiKill = "a Pentakill";
+						break;
+					case 6: multiKill = "a Hexakill";
+						break;
+					case 7: multiKill = "a Septakill";
+						break;
+					case 8: multiKill = "an Octakill";
+						break;
+					case 9: multiKill = "a Nonakill";
+						break;
+					case 10: multiKill = "a Decakill";
+						break;
+					case 11: multiKill = "an " + blob.get_u8("killstreak") + " kill multikill";
+						break;
+					case 18: multiKill = "an " + blob.get_u8("killstreak") + " kill multikill";
+						break;
+					default: multiKill = "a " + blob.get_u8("killstreak") + " kill multikill";
+						break;
+				}
+
+				client_AddToChat(player.getCharacterName() + " got " + multiKill + "!", SColor(255, 180, 24, 94));
+					
+				blob.set_u8("killstreak", 0);
```

Comment:
this should be moved in a `isServer()` branch, and it should be `Sync`'d like it is in `onPlayerDie`